github/issues.go

```diff
@@ -44,6 +44,7 @@ type Issue struct {
 	Repository       *Repository       `json:"repository,omitempty"`
 	Reactions        *Reactions        `json:"reactions,omitempty"`
 	Assignees        []*User           `json:"assignees,omitempty"`
+	ClosedBy         *User             `json:"closed_by,omitempty"`
```

Comment:
Alternatively, you can place it under `UpdatedAt`, so that it's consistent with the example GitHub API docs use:

![image](https://cloud.githubusercontent.com/assets/1924134/23584327/4731077e-012c-11e7-8f1d-6a9727ef37dc.png)